bin/create_k8s_token.sh

```diff
@@ -0,0 +1,124 @@
+#!/usr/bin/env bash
+
```

Comment:
agreed, wanted to keep it as light weight as possible since it's running via cron.